packages/styles/scss/components/data-table/_data-table-action.scss

```
@@ -0,0 +1,586 @@
+//
+// Copyright IBM Corp. 2016, 2018
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+@use '../../config' as *;
+@use '../../motion' as *;
+@use '../../spacing' as *;
+@use '../../theme' as *;
+@use '../../type' as *;
+@use '../../utilities/button-reset';
+@use '../../utilities/convert' as *;
+@use '../../utilities/focus-outline' as *;
+
+/// Data table action styles
+/// @access private
+/// @group data-table
+@mixin data-table-v2-action {
```

Just a refactor note for any v2 references we still have leftover 👀 

```suggestion
@mixin data-table-action {
```

---

Fixed 👍